partcad/src/partcad/assembly.py

```
@@ -27,11 +28,12 @@ def __init__(self, item, name=None, location=None):
 class Assembly(ShapeWithAi):
     path: typing.Optional[str] = None
 
-    def __init__(self, project_name: str, config: dict = {}):
-        super().__init__(project_name, config)
+    def __init__(self, project_name: str, config: dict = {}, user_config: UserConfig = None):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)